src/core_ocean/analysis_members/Registry_lagrangian_particle_tracking.xml

```diff
@@ -316,11 +324,17 @@
 			 description="flag to monitor if the particle was transfered"
 		/>
 <!-- TEMP FIELDS THAT NEED MOVED TO NonHalo below after verifying interpolation is correct -->
-		<var name="temperature" type="real" dimensions="nParticles Time" units="kg m^{-3}"
+		<var name="particleTemperature" type="real" dimensions="nParticles Time" units="degrees Celsius"
```

Comment:
Here is the example referenced in the comment below-- `temperature` got renamed to `particleTemperature` but this wasn't reflected below in the call to grab the array.